src/Compilers/CSharp/Test/Semantic/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -4934,18 +4954,18 @@ public static void Main()
             Assert.Empty(flowAnalysis.Captured);
             Assert.Empty(flowAnalysis.CapturedInside);
             Assert.Empty(flowAnalysis.CapturedOutside);
-            Assert.Equal("MyClass @this", flowAnalysis.DataFlowsIn.Single().ToTestDisplayString());
+            Assert.Equal("MyClass this", flowAnalysis.DataFlowsIn.Single().ToTestDisplayString());
```

Comment:
yup.  these are all bad changes.

Reply:
No, I think those changes are correct. Added some explicit tests with "this" and "@this" to illustrate the point.

---
In reply to: [160014222](https://github.com/dotnet/roslyn/pull/23970#discussion_r160014222) [](ancestors = 160014222)

Reply:
Gotcha!